munch/__init__.py

```diff
@@ -21,11 +21,20 @@
     converted via Munch.to/fromDict().
 """
 
-import pkg_resources
+try:
+    # For python 3.8 and later
+    import importlib.metadata as importlib_metadata
+except ImportError:
+    # For everyone else
+    import importlib_metadata
+try:
+    __version__ = importlib_metadata.version(__name__)
+except importlib_metadata.PackageNotFoundError:
+    # package is not installed
+    pass
 
 from .python3_compat import iterkeys, iteritems, Mapping, u
```

Comment:
Can you move this to the top of the module so that the linter won't fail? 
Thanks

Reply:
Done